qiskit/pulse/channels/pulse_channel.py

```diff
@@ -54,14 +54,29 @@ def __init__(self, index):
         super().__init__(index)
 
 
-class SnapshotChannel(PulseChannel):
```

Comment:
Why was SnapshotChannel deleted?

Reply:
Excuse me. I've looked too narrow. I'll restore it.

Reply:
done